packages/api/core/src/api/start.ts

```diff
@@ -153,6 +153,8 @@ export default async ({
       } as NodeJS.ProcessEnv,
     };
 
+    delete spawnOpts.env.NODE_OPTIONS;
```

Comment:
I would like this to conditionally delete the strings that Yarn adds from `process.env.NODE_OPTIONS` instead of clearing the entire env var.

Unfortunately, `commander` doesn't seem to expose any utilities to parse args strings, and `util.parseArgs` is only available in Node 18+, so we might need to just hack this in manually.